mne/epochs.py

```diff
@@ -1810,6 +1810,56 @@ def save(self, fname, split_size='2GB', fmt='single', overwrite=False,
             this_epochs.event_id = self.event_id
             _save_split(this_epochs, fname, part_idx, n_parts, fmt)
 
+    def export(self, fname, fmt="auto"):
+        """Export Epochs to EEGLAB's .set format.
+
+        Parameters
+        ----------
+        fname : str
+            Name of the export file.
+        fmt : str
+            Format of the export. Defaults to auto, which will infer
+            format from fname's extension
```

Comment:
thinking ahead to brainvision exporting:  doesn't brainvision write *three* files (.eeg, .vmrk, .vhdr)?  How will this API accommodate that?  I'd rather get it right now than have to change it later.

Reply:
I'd accept all three extensions for BrainVision.